torch_frame/data/mapper.py

```diff
@@ -301,19 +301,30 @@ def forward(
         ser: Series,
         *,
         device: Optional[torch.device] = None,
-    ) -> Tensor:
+    ) -> MultiEmbeddingTensor:
         ser = ser.astype(str)
         ser_list = ser.tolist()
         if self.batch_size is None:
             emb = self.text_embedder(ser_list)
-            return emb.to(device)
+            return MultiEmbeddingTensor(
+                num_rows=len(ser),
+                num_cols=1,
+                values=emb,
+                offset=torch.tensor([0, len(emb[0])]),
+            ).to(device)
 
         emb_list = []
         for i in tqdm(range(0, len(ser_list), self.batch_size),
                       desc="Embedding texts in mini-batch"):
             emb = self.text_embedder(ser_list[i:i + self.batch_size])
             emb_list.append(emb.to(device))
-        return torch.cat(emb_list, dim=0)
+        emb = torch.cat(emb_list, dim=0)
+        return MultiEmbeddingTensor(
+            num_rows=len(ser),
+            num_cols=1,
+            values=emb,
+            offset=torch.tensor([0, len(emb[0])]),
+        ).to(device)
```

Comment:
There are some code repetition in `MultiEmbeddingTensor` construction.
Can we do
```python
if self.batch_size is None:
   emb = ...
else:
   ...
   emb = torch.cat(emb_list, dim=0)
return MultiEmbeddingTensor(..., values = emb, ...)
```